commands/post.go

```diff
@@ -38,13 +39,19 @@ var PostListCmd = &cobra.Command{
 	RunE: withClient(postListCmdF),
 }
 
+const (
+	ISO8601Layout  = "2006-01-02T15:04:05-07:00"
```

Comment:
The original request only calls for supporting this timestamp, so it works for me. @isacikgoz I'm not as familiar with expectations on time formats for mmctl. Does only supporting ISO8601 sound reasonable?

Reply:
We can add more time formats later if requested, ISO8601 is a common standard so it's good to me.